srcCpp/RTIDDSImpl.cxx

```diff
@@ -92,7 +96,7 @@ DynamicDataMembersId &DynamicDataMembersId::GetInstance()
 
 int DynamicDataMembersId::at(std::string key)
 {
-   return membersId.at(key);
+   return membersId[key];
```

Comment:
I foresee that this could lead us to some issues if people are using 2.4 out there. What do you think if we add this to master separately (as an independent commit, also for C++11) with the documentation, in case we want to cherry-pick it to the `release/2.4`?

Reply:
Full agree
Done